src/static/js/templates/prepare-worksheets/worksheet-goals.hbs

```diff
@@ -0,0 +1,18 @@
+<!-- Goals -->
+<h2><span class="cf-icon cf-icon-approved-round"></span> Clarify your goals</h2>
+
+<p class="lead">Most people are interested in buying a home for a mix of lifestyle and financial reasons. Make a list of all the reasons you’re interested in buying a home. <em>We’ve listed some common ones to get you started – edit the items or delete ones that are not relevant to you, and add your own item.</em></p>
+
+<!-- Lifestyle Goals -->
+<section class="worksheet worksheet-goal">
+  <header class="subheading">
+    <h4 class="u-right">Priority level</h4>
```

Comment:
I would recommend using `aria-hidden` to keep these from getting read by screen readers. I don't think it would make sense to hear these two headings read out. The `<label>`'s from my previous comment replace the need for these anyway.